src/Microsoft.Health.Fhir.Core/Models/SearchParameterInfo.cs

```diff
@@ -19,10 +19,27 @@ public class SearchParameterInfo
             IReadOnlyList<SearchParameterComponentInfo> components = null,
             string expression = null,
             IReadOnlyCollection<string> targetResourceTypes = null)
+            : this(
+                name,
+                url,
+                Enum.TryParse<SearchParamType>(searchParamType, out var type) ? (SearchParamType?)type : null,
+                components,
+                expression,
+                targetResourceTypes)
+        {
+        }
+
+        public SearchParameterInfo(
```

Comment:
When would we have SearchParameterInfo without url, type, and expresion and etc defined?

Reply:
There are a few tests that do this. The original model didn't validate against being empty